src/reduxStore/store.ts

```diff
@@ -29,22 +27,18 @@ const migrations = {
   }
 };
 
-//This allows for this library to be used on other platforms than web, like React Native
-//without this condition, redux-persist 6+ will throw an error if persist storage fails
-if (typeof window !== 'undefined' && window?.localStorage != null) {
-  const storage = require('redux-persist/lib/storage').default;
-  const persistConfig = {
-    key: 'dapp-core-store',
-    version: 2,
-    storage,
-    whitelist: ['account', 'loginInfo', 'toasts', 'modals', 'networkConfig'],
-    migrate: createMigrate(migrations, { debug: false })
-  };
-  localStorageReducers = persistReducer(persistConfig, rootReducer);
-}
+const persistConfig: any = {
+  key: 'dapp-core-store',
+  version: 2,
+  storage,
+  whitelist: ['account', 'loginInfo', 'toasts', 'modals', 'networkConfig'],
```

Comment:
Strings could be defined under constants/whitelisdedSlices and imported from there, to avoid typos